airflow/contrib/operators/ecs_operator.py

```diff
@@ -73,6 +73,8 @@ class ECSOperator(BaseOperator):
     :type platform_version: str
     :param network_configuration: the network configuration for the task
     :type network_configuration: dict
+    :param tags: a list of tags in the form of [{'key': 'tagKey', 'value': 'tagValue'}].
+    :type tags: list
```

Comment:
I know this is how the boto API accepts tags, but might it be easier for users to instead accept a dict as `{'tagKey':, 'tagValue'}`?

Reply:
yup, sounds good, changed it